microprofile/opentracing-extension/src/test/java/org/wildfly/extension/microprofile/opentracing/Subsystem_2_0_ParsingTestCase.java

```diff
@@ -152,6 +152,7 @@ protected void initializeExtraSubystemsAndModel(ExtensionRegistry extensionRegis
             super.initializeExtraSubystemsAndModel(extensionRegistry, rootResource, rootRegistration, capabilityRegistry);
             Map<String, Class> capabilities = new HashMap<>();
             capabilities.put(WELD_CAPABILITY_NAME, WeldCapability.class);
+            capabilities.put("org.wildfly.microprofile.config", Void.class);
```

Comment:
Please use org.wildfly.extension.microprofile.opentracing.SubsystemDefinition#MICROPROFILE_CONFIG_CAPABILITY_NAME instead and make it public.